src/main/java/seedu/address/model/transaction/Amount.java

```diff
@@ -40,6 +40,12 @@ public Amount add(Amount amountToAdd) {
         return new Amount(totalAmount);
     }
 
+    /**
+     * Returns true if the amount is zero.
+     */
+    public boolean isZero() {
```

Comment:
Might not work with double, can check isZero using a threshold value e.g. abs(amount - 0) <threshold or maybe we should change implementation of Amount to not use double